kokoro/continuous.sh

```diff
@@ -3,14 +3,17 @@
 set -e
 set -x
 
-gcloud components install docker-credential-gcr
+# On Mac, the default "credsStore" is set to "desktop". However, "desktop" is a
+# protected credential store, so "docker login" fails to modify it.
+# https://github.com/GoogleContainerTools/jib/issues/2189
+if [ "${KOKORO_JOB_CLUSTER}" = "MACOS_EXTERNAL" ]; then
```

Comment:
should we file a bug against kokoro? or is that working as intended?

Reply:
I don't actually know, but perhaps it's a sane request to do something about this, so that `docker login` just works. I'll file a bug.